.travis.yml

```diff
@@ -1,75 +1,76 @@
-sudo: false

```

Comment:
Not sure what happened here: was the file commited with `\r\n` before?

Reply:
No idea how this happened